DNN Platform/DotNetNuke.Abstractions/Portals/IPortalAliasInfo.cs

```diff
@@ -1,27 +1,59 @@
 ﻿// Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information
+
 namespace DotNetNuke.Abstractions.Portals
 {
-    using System.Data;
-    using System.Xml;
-    using System.Xml.Schema;
+    using DotNetNuke.Abstractions.Urls;
 
+    /// <summary>
+    /// Portal Alias Info interface.
+    /// </summary>
     public interface IPortalAliasInfo
     {
-        //BrowserTypes BrowserType { get; set; }
-        string CultureCode { get; set; }
+        /// <summary>
+        /// Gets or sets the http alias.
+        /// </summary>
         string HTTPAlias { get; set; }
-        bool IsPrimary { get; set; }
-        int KeyID { get; set; }
+
+        /// <summary>
+        /// Gets or sets the Portal Alias ID.
+        /// </summary>
         int PortalAliasID { get; set; }
+
+        /// <summary>
+        /// Gets or sets the Portal ID.
+        /// </summary>
         int PortalID { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether if the portal is the primary portal.
+        /// </summary>
+        bool IsPrimary { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether the poral is a redirect portal.
+        /// </summary>
         bool Redirect { get; set; }
+
+        /// <summary>
+        /// Gets or sets the Browser Type.
+        /// </summary>
+        BrowserTypes BrowserType { get; set; }
+
+        /// <summary>
+        /// Gets or sets the culture code.
+        /// </summary>
+        string CultureCode { get; set; }
+
+        /// <summary>
+        /// Gets or sets the skin.
+        /// </summary>
         string Skin { get; set; }
 
-        void Fill(IDataReader dr);
-        XmlSchema GetSchema();
-        void ReadXml(XmlReader reader);
-        void WriteXml(XmlWriter writer);
```

Comment:
These APIs should never have been added to the `IPortalAliasInfo` in the first place. I removed them as they are contained within other .NET System level interfaces. This is technically a breaking change and I am not sure how we best manage this.

@bdukes @mitchelsellers any suggestions on this?

Reply:
Not saying it's not a breaking change but it was added in 9.7.0 and I doubt many 3rd parties call this... IMHO, I vote for removing it quick rather that keeping it in there deprecated for 2 full versions.

Reply:
I think that this will be an acceptable break due to the limited time, as users shouldn't be using it so that is the only time it would be truly breaking

Reply:
I am marking this as resolved as we are deleting the items from the `IHydratable` interface in the abstractionsp roject